src/main/java/sx/blah/discord/handle/audit/entry/change/ChangeKey.java

```diff
@@ -0,0 +1,92 @@
+	public static final ChangeKey<String> ICON_HASH = newKey();
+	public static final ChangeKey<String> SPLASH_HASH = newKey();
+	public static final ChangeKey<Long> OWNER_ID = newKey();
+	public static final ChangeKey<String> REGION = newKey();
+	public static final ChangeKey<Long> AFK_CHANNEL_ID = newKey();
+	public static final ChangeKey<Integer> AFK_TIMEOUT = newKey();
+	public static final ChangeKey<Integer> MFA_LEVEL = newKey();
+	public static final ChangeKey<Integer> VERIFICATION_LEVEL = newKey();
+	public static final ChangeKey<Integer> EXPLICIT_CONTENT_FILTER = newKey();
+	public static final ChangeKey<Integer> DEFAULT_MESSAGE_NOTIFICATIONS = newKey();
+	public static final ChangeKey<String> VANITY_URL_CODE = newKey();
+	public static final ChangeKey<IRole[]> $ADD = newKey();
+	public static final ChangeKey<IRole[]> $REMOVE = newKey();
+	public static final ChangeKey<Integer> PRUNE_DELETE_DAYS = newKey();
+	public static final ChangeKey<Boolean> WIDGET_ENABLED = newKey();
+	public static final ChangeKey<Long> WIDGET_CHANNEL_ID = newKey();
+	public static final ChangeKey<Integer> POSITION = newKey();
+	public static final ChangeKey<String> TOPIC = newKey();
+	public static final ChangeKey<Integer> BITRATE = newKey();
+	public static final ChangeKey<Boolean> NSFW = newKey();
+	public static final ChangeKey<Long> APPLICATION_ID = newKey();
+	public static final ChangeKey<Integer> PERMISSIONS = newKey();
+	public static final ChangeKey<Integer> COLOR = newKey();
+	public static final ChangeKey<Boolean> HOIST = newKey();
+	public static final ChangeKey<Boolean> MENTIONABLE = newKey();
+	public static final ChangeKey<Integer> ALLOW = newKey();
+	public static final ChangeKey<Integer> DENY = newKey();
+	public static final ChangeKey<String> CODE = newKey();
+	public static final ChangeKey<Long> CHANNEL_ID = newKey();
+	public static final ChangeKey<Long> INVITER_ID = newKey();
+	public static final ChangeKey<Integer> MAX_USES = newKey();
+	public static final ChangeKey<Integer> USES = newKey();
+	public static final ChangeKey<Integer> MAX_AGE = newKey();
+	public static final ChangeKey<Boolean> TEMPORARY = newKey();
+	public static final ChangeKey<Boolean> DEAF = newKey();
+	public static final ChangeKey<Boolean> MUTE = newKey();
+	public static final ChangeKey<String> NICK = newKey();
+	public static final ChangeKey<String> AVATAR_HASH = newKey();
+	public static final ChangeKey<Long> ID = newKey();
+	public static final ChangeKey<Object> TYPE = newKey();
+	public static final ChangeKey<IChannel.PermissionOverride[]> PERMISSION_OVERWRITES = newKey();
+
+	public static ChangeKey<?> fromRaw(String rawKey) {
+		ChangeKey<?> key = null;
+		try {
+			key = (ChangeKey) ChangeKey.class.getDeclaredField(rawKey.toUpperCase()).get(null);
+		} catch (NoSuchFieldException | IllegalAccessException e) {
+			Discord4J.LOGGER.error(LogMarkers.HANDLE, "Failed to find change key from raw string: {}", rawKey, e);
```

Comment:
I think since the input to the method is directly from Discord, it should just hard fail. If two unknown keys were used, the collector would fail on merging. It would also be odd to be allowed to do something like `getChangeByKey(ChangeKey.UNKNOWN)`